config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
 
   namespace :v1, defaults: { format: 'json' } do
+    post 'user_token' => 'user_token#create'
```

Comment:
What do you thing about a more generic resource here?  Maybe something not tied directly to the user resource, like `auth/token`, `authentications/token`,  or just have `authorize` map to a post action?